src/components/grid/grid-body.tsx

```diff
@@ -73,6 +76,20 @@ export const GridBody: React.FC<GridBodyProps> = ({
         className={styles.gridTick}
       />
     );
+
+    if (weekendColor !== "transparent" && dates[i + 1] && [0,6].includes(dates[i + 1].getDay())) {
```

Comment:
Curious, how did you test this ? 

what are the steps to run the ./example with this change ?